### DISCUSSION TITLE
Inverse problem error

### DISCUSSION THREAD

**Discussion by nirmal129** [lululxvi/deepxde#1304] [Q&A] [answered]
Hello, 

I am trying to solve an inverse wave equation where I have data and need to find the velocity. I am referring to the code on the documentation to solve the problem. However, I am receiving a series of exceptions, namely 'Size does not work for tensors > int32 max' and 'Graph execution error,' and I have no clue how to approach them. I am attaching my code file and dataset in zip format below. The code file is written for Google Colab, however, small changes will make it work on Jupyter Notebook. 

[Archive.zip](https://github.com/lululxvi/deepxde/files/11426696/Archive.zip)

I appreciate your efforts. 

Thank you! 
Nirmal Patel

EDIT2: I removed my pasted error because I found the issue. I found that I was getting the error because my dataset was too large. The dataset is of a similar format to what's given in the documentation, i.e. XT and y_data. What's the maximum length of XT (or y_data, doesn't matter as both should be of the same length) allowed in inverse PINN to avoid the error 'Size does not work for tensors > int32 max'?

**Comment by lululxvi** [comment]
https://learn.microsoft.com/en-us/dotnet/api/system.int32.maxvalue?view=net-7.0

**Reply by nirmal129** [reply]
The original dataset (given in the zip file) contains less than a million data points. Int32 max is about 2 billion. That's why I am confused why the inverse PINN model won't run even if the number of data points is magnitudes less than int32 max.

**Reply by lululxvi** [reply, marked as answer]
There could be other overhead.
